package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "combinations",
-  "version": "0.1.1",
```

Comment:
I've changed the version as NPM don't check the changes...
[DOCS](https://docs.npmjs.com/files/package.json#version)